code/src/components/Button.js

```diff
@@ -0,0 +1,73 @@
+import styled, { css } from 'styled-components';
+
+export const Button = styled.button`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  border: none;
+  border-radius: 50px;
+
```

Comment:
A tiny tip (you might know this but decided not to use it, sorry for stating the obvious if so!), you could use `cursor: pointer;` or some other cursor choice if you want on your buttons to clarify that it is a clickable object for the user. Not a must just a "nice to have" if one likes it.  (https://www.w3schools.com/cssref/pr_class_cursor.php)